openproblems/tasks/denoising/methods/knn_smoothing.py

```diff
@@ -8,16 +8,17 @@
 
 # the decay and t parameter give knn smoothing behavior
 def _knn_smoothing(adata, solver):
-    import knn-smooth

```

Comment:
Package names can't have `-` in them. It should be `knn_smooth`

Reply:
fixed this (silly me)